realm/realm-library/src/objectServer/java/io/realm/internal/network/StreamNetworkTransport.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.internal.network;
+
+import org.bson.BsonArray;
+
+import java.io.IOException;
+
+import io.realm.internal.objectstore.OsJavaNetworkTransport;
+
+public abstract class StreamNetworkTransport {
```

Comment:
I would still add a bit of documentation, mostly stating why we need this and what it does. It shouldn't be extensive since it's for internal use.